data/ppdb-ap-pipe-schema-extra.yaml

```diff
@@ -18,6 +18,419 @@ columns:
   nullable: false
   description: Total number of DiaSources associated with this DiaObject.
 
+- name: uPSFluxMedian
+  type: FLOAT
+  nullable: true
+  description: Median u band flux.
+  unit: nJy
+- name: uPSFluxMAD
+  type: FLOAT
+  nullable: true
+  description: Median absolute deviation u band fluxes.
+  unit: nJy
+- name: uPSFluxSkew
+  type: FLOAT
+  nullable: true
+  description: Skewness of the u band fluxes.
+- name: uPSFluxPercentile05
+  type: FLOAT
+  nullable: true
+  description: Value at the 5% percentile of the u band fluxes.
+  unit: nJy
+- name: uPSFluxPercentile25
+  type: FLOAT
+  nullable: true
+  description: Value at the 25% percentile of the u band fluxes.
+  unit: nJy
+- name: uPSFluxPercentile75
+  type: FLOAT
+  nullable: true
+  description: Value at the 75% percentile of the u band fluxes.
+  unit: nJy
+- name: uPSFluxPercentile95
+  type: FLOAT
+  nullable: true
+  description: Value at the 95% percentile of the u band fluxes.
+  unit: nJy
+- name: uPSFluxMin
+  type: FLOAT
+  nullable: true
+  description: Minimum observed u band fluxes.
+  unit: nJy
+- name: uPSFluxMax
+  type: FLOAT
+  nullable: true
+  description: Maximum observed u band fluxes.
+  unit: nJy
+- name: uPSFluxStetsonJ
+  type: FLOAT
+  nullable: true
+  description: StetsonJ statistic for the u band fluxes.
+- name: uPSFluxLinearSlope
+  type: FLOAT
+  nullable: true
+  description: Linear best fit slope of the u band fluxes.
+  unit: nJy/MJD
```

Comment:
should be `nJy/day` (MJD is a time system, not a time unit).  Same for the other slopes here.